05-analog-signage/firmware/code.py

```diff
@@ -27,7 +27,7 @@
 i2c = board.I2C()
 card = notecard.OpenI2C(i2c, 0, 0, debug=False)
 
-req = {"req": "hub.set", "mode": "continuous", "sync": True}
+req = {"req": "hub.set", "mode": "continuous", "sync": True, "body": {"app":"nf5"}}
```

Comment:
This is not correct. `note-python` does support the user-agent info, so I need to rework this (and test.)

Reply:
`note-python` updated with user agent support.